pkgs/development/python-modules/pycryptotools/default.nix

```
@@ -0,0 +1,42 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  pbkdf2,
+  requests,
+  cashaddress,
+  pyperclip,
+  setuptools,
+  base58,
+}:
+
+buildPythonPackage rec {
+  pname = "pycryptotools";
+  version = "0.3.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-oXvBit2Z8o6QXC12ob1jX9A0wsev6Kfbtwz/fBmnRls=";
+  };
+
+  propagatedBuildInputs = [
+    pbkdf2
+    requests
+    cashaddress
+    pyperclip
+    base58
+  ];
+
+  pyproject = true;
+  build-system = [ setuptools ];
+
+  pythonImportsCheck = [ "pycryptotools" ];
+
+  meta = with lib; {
+    changelog = "https://pypi.org/project/pycryptotools/#history";
+    description = "Pycryptotools, Python library for Crypto coins signatures and transactions";
+    homepage = "https://pypi.org/project/pycryptotools/";
+    license = licenses.mit;
+    maintainers = with maintainers; [ eymeric ];
+  };
```

Lets keep `with lib;` on small expression, to avoid its caveats
```suggestion
  meta = {
    changelog = "https://pypi.org/project/pycryptotools/#history";
    description = "Pycryptotools, Python library for Crypto coins signatures and transactions";
    homepage = "https://pypi.org/project/pycryptotools/";
    license = lib.licenses.mit;
    maintainers = with lib.maintainers; [ eymeric ];
  };
```